### PR TITLE
Avoid sending Compact request when LazyFS is enabled

### PR DESCRIPTION
**Pull request by serathius**
For https://github.com/etcd-io/etcd/issues/18144

cc @siyuanfoundation

### CODE REVIEW

File: tests/robustness/traffic/etcd.go [hunk 1]
```diff
@@ -72,6 +72,21 @@ type etcdTraffic struct {
 	largePutSize int
 }
 
+func (t etcdTraffic) WithoutRevision() Traffic {
```
Comment (siyuanfoundation) [inline tests/robustness/traffic/etcd.go:75]: why is this called `WithoutRevision` while it seems to be `WithoutCompact`?

Comment (serathius) [reply tests/robustness/traffic/etcd.go:75]: Ups :P

**Review by siyuanfoundation — APPROVED** [review]

**Comment by serathius** [comment]
cc @fuweid @MadhavJivrajani

File: tests/robustness/scenarios.go [hunk 2]
```diff
@@ -141,6 +131,23 @@ func exploratoryScenarios(_ *testing.T) []testScenario {
...
 	}
+	if e2e.BinPath.LazyFSAvailable() {
+		newScenarios := scenarios
+		for _, s := range scenarios {
+			// LazyFS increases the load on CPU, so we run it with more lightweight case.
+			if s.profile.MinimalQPS <= 100 && s.cluster.ClusterSize == 1 {
+				newScenarios = append(newScenarios, testScenario{
+					name:      filepath.Join(s.name, "LazyFS"),
+					failpoint: s.failpoint,
+					cluster:   s.cluster,
```
Comment (fuweid) [inline tests/robustness/scenarios.go:142]: it seems we loss `e2e.WithLazyFSEnabled(true)` here?

Comment (serathius) [reply tests/robustness/scenarios.go]: Ups again.

**Pull request merged**